models/org_team.go

```diff
@@ -309,14 +314,20 @@ func NewTeam(t *Team) (err error) {
 			unit.TeamID = t.ID
 		}
 		if _, err = sess.Insert(&t.Units); err != nil {
-			sess.Rollback()
+			err = sess.Rollback()
+			if err != nil {
+				return err
+			}
 			return err
 		}
 	}
 
 	// Update organization number of teams.
 	if _, err = sess.Exec("UPDATE `user` SET num_teams=num_teams+1 WHERE id = ?", t.OrgID); err != nil {
-		sess.Rollback()
+		err = sess.Rollback()
+		if err != nil {
```

Comment:
This if statement is not necessary because no matter the outcome err will be returned